kratos/geometries/geometry.h

```diff
@@ -555,26 +559,55 @@ class Geometry
             *i = typename PointType::Pointer( new PointType( **i ) );
     }
 
-     // virtual Kratos::shared_ptr< Geometry< Point > > Clone() const
-     // {
-     //     Geometry< Point >::PointsArrayType NewPoints;
+    ///@}
+    ///@name Id
+    ///@{
+
+    /// Id of this Geometry
+    IndexType Id() const
+    {
+        KRATOS_ERROR_IF(HasGeometryIdString())
+            << "Assigned geometry id is name of type string."
+            << std::endl;
+
+        return mId;
+    }
+
+    /// Id of this Geometry
+    IndexType GetHashId() const
```

Comment:
This would GetId method, like the index object

Reply:
I changed it accordingly. Thx